src/__tests__/components/Routes.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
What if you rename this file UserRoutes.test.js or put it in user folder so that if teammates introduce Unit testing for their routes, won't cause conflicts?